test/core/DependencyUnitTests/CMakeLists.txt

```diff
@@ -547,7 +547,8 @@ create_dependency_handling_test_case(
   ARGS
     -DTrilinos_ENABLE_ALL_PACKAGES:BOOL=ON
     -DTrilinos_DUMP_PACKAGE_DEPENDENCIES:BOOL=ON
--DTrilinos_WARN_ABOUT_MISSING_EXTERNAL_PACKAGES=ON
+    -DTrilinos_DUMP_FORWARD_PACKAGE_DEPENDENCIES:BOOL=ON
+    -DTrilinos_WARN_ABOUT_MISSING_EXTERNAL_PACKAGES=ON
```

Comment:
**Question:** Should there be a `:BOOL` here?

Reply:
For consistency, sure.  But as we know, no behavior is altered at all for types like `BOOL` (unlike `PATH` and `FILEPATH`).

Reply:
Resolved in new commit 515df6c7.